### PR TITLE
86 improve readme

### PR DESCRIPTION
**Pull request by Juni-hub**
Improved main readme with more text, pictures and code coverage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,85 @@
...
+  </div>
+  <div class="column">
+
+  </div>
+</div>
+<p float="left">
+    <a href="https://revolve.no/"><img align=left src="https://raw.githubusercontent.com/RevolveNTNU/RosNet/86-improve-readme/.github/main/revolve.svg" width="30%" height="100"/></a>
+    <a href="https://ros.org/"><img align=right margin src="https://raw.githubusercontent.com/RevolveNTNU/RosNet/86-improve-readme/.github/main/ros.png" height="100"/></a>
+</p>
+<br>
```
Comment (henrikhorluck) [inline README.md:13]: One `<br>` should be enough?

File: README.md [hunk 2]
```diff
@@ -1,26 +1,85 @@
...
 ```
-For full API documentation go to: https://revolventnu.github.io/RosNet/
 
+### FieldValue
+
+FieldValue is a custom data type with a 
+
+- string name -  name of the field
+- PrimitiveType DataType -  datatype of the value. PrimitiveType is an Enum with possible values: Bool, Int8, Uint8, Int16, Unit16, Int32, Uint32, Int64, Uint64, Float32, Float64, String, Time, Duration, Byte, Char and Array. The datatypes correspond to the [standard datatypes](https://wiki.ros.org/msg) in ROS messages and Array used for arrays of values.
+- byte[] Value - is the value of the field
```
Comment (henrikhorluck) [inline README.md:55]:
```suggestion
You will primarily interact with `FieldValue` which has the following fields

- `string Name` -  name of the field
- `PrimitiveType DataType` -  datatype of the value. PrimitiveType is an Enum with possible values: Bool, Int8, Uint8, Int16, Unit16, Int32, Uint32, Int64, Uint64, Float32, Float64, String, Time, Duration, Byte, Char and Array. The datatypes correspond to the [standard datatypes](https://wiki.ros.org/msg) in ROS messages and Array used for arrays of values.
- `byte[] Value` - is the value of the field
```

Calling it a "custom data type" is not necessary, and could rather be a tiny bit confusing with current development of e.g. data classes in Python, (usually called records in C# lingo though)

File: RosNet.Test/RosNet.Test.csproj [hunk 3]
```diff
@@ -5,6 +5,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="coverlet.msbuild" Version="3.0.3">
```
Comment (henrikhorluck) [inline RosNet.Test/RosNet.Test.csproj:8]: I don't think this is necessary?

**Review by henrikhorluck — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Juni-hub) [reply README.md:13]: The badges and title will come inbetween the pictures if not

File: RosNet.Test/RosNet.Test.csproj (hunk 3 above)
Comment (Juni-hub) [reply RosNet.Test/RosNet.Test.csproj:8]: the coverlet github action could not find the coverage file without it

**Pull request merged**